src/components/CreateUser/CreateUser.scss

```diff
@@ -37,13 +41,18 @@
     background-color: #4caf50;
```

Comment:
Can we use a css variable from the constants file on line 41 as well?

Reply:
changed